sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/HttpTimeoutPolicyControlPlaneHotPath.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.http;
+
+import com.azure.cosmos.implementation.Configs;
+import com.azure.cosmos.implementation.RxDocumentServiceResponse;
+import io.netty.handler.codec.http.HttpMethod;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static com.azure.cosmos.implementation.HttpConstants.StatusCodes.REQUEST_TIMEOUT;
+
+public class HttpTimeoutPolicyControlPlaneHotPath extends HttpTimeoutPolicy {
+
+    final public static HttpTimeoutPolicy instance = new HttpTimeoutPolicyControlPlaneHotPath();
```

Comment:
nit: naming convention for final static variables. 
```suggestion
    final public static HttpTimeoutPolicy INSTANCE = new HttpTimeoutPolicyControlPlaneHotPath();
```

Reply:
Fixed it.